internal/server/admin/schemas_service.go

```diff
@@ -174,7 +174,10 @@ func (s *SchemasService) validateSchema(ctx context.Context, item proto.Message)
 		// registered (via `model.RegisterType()`), this would never error.
 		return err
 	}
-
+	// Process entity defaults before performing schema validation.
+	if err := obj.ProcessDefaults(ctx); err != nil {
+		return err
```

Comment:
I have some work tracked to remove this silly error handling, but for now, we need to be very explicit:
```suggestion
		return s.err(ctx, err)
```

Reply:
done

Reply:
for my understanding, what is the impact of not using that handler? 

I also see it is now used here as well: https://github.com/Kong/koko/pull/491/files#diff-f4d96c10298118aa8c7beeb59e9d437aa9438651e7bde8aebd6870d546ea19a1R175 (not part of this PR)

Reply:
If that handler isn't used in the applicable code path, the actual error will not be relayed to the end-user. E.g.: If that particular code exposes an error that end-users need to care about, it'd need to be wrapped with that error handler.

The code you linked @aboudreault wouldn't be an error an end-user needs to see.

Reply:
got it. thanks @tjasko!